### ISSUE TITLE
Investigate native video player screen reader compatibility in Edge

### ISSUE THREAD

**Issue by nickcolley** [alphagov/govuk-frontend#1425]
## What

We've had reports that the native video players we're using on the design system website are not accessible to screen reader users: https://github.com/alphagov/govuk-design-system/issues/904

## Why

Make want to sure that browser vendors are aware of the issues so they can be resolved.

## Done when

- [ ] Check if Microsoft Edge (including the chromium version) has an accessible native player
- [ ] If there are issues raise an issue with the browser vendor
